### ISSUE TITLE
Issue: Change the fan speed in the middle of a cook then shut down the smoker.  Start another cook and the fan does not start until rebooting the controller.

### ISSUE THREAD

**Issue by muxall** [muxall/RPTCQ-LCD#123]
Shawn reported:

If I change the speed in the middle of a cook, then shut down the smoker, when I go to use again, the fan does not start until I reboot the controller.
the ignition rod works, but no fan. Even if you manually change the fan settings, it has no affect.

**Comment by muxall** [comment]
Shawn reports: I ran another test.. After modifying the fan speed during a cook, when you start another cook in the app the fan will not turn on unless you power cycle the controller.
That is cook, with full shutdown and the next day starting a new cook.

**Comment by muxall** [comment]
Bernard reported: I ran into a problem with the Muxall today where the fan wouldn't turn on when I started a cook earlier this afternoon.  In the web app it showed fan = 0%. When I set mode = off, it also didn't respond to a manual override to force the fan on.  I did see voltage coming from the board to the fan, according to my contactless voltage tester.  Interestingly, when I shut off (using toggle switch), then turn back on, I was able to control the fan manually using override.  However, if I shut it off within the web app, then it would not turn back on until I did a hard reset using the toggle switch.  I then did a hard reset using the toggle switch and started a cook, which for some reason turned on  the fan.  Am I looking at a software bug here?  If it was hardware related, I would have expected the status to show fan = 100% because it was "trying" to control the fan.  Wonder if you've seen this before?
